### ISSUE TITLE
Install fails due to cffi==1.14.5 dependency not being met

### ISSUE THREAD

**Issue by boonhapus** [thoughtspot/cs_tools#21]
### First Stop

- [X] I have read and agreed to the ThoughtSpot Dev Tools EULA.
- [X] I understand that if I have a critical issue with ThoughtSpot, I should reach out to my account team instead of cs_tools.
- [X] I searched the cs_tools documentation, with the integrated search.

### Platform Configuration

```
(.cs_tools) C:\work\thoughtspot\cs_tools>cstools platform

        [PLATFORM DETAILS]
        system: Windows (detail: Windows-10-10.0.19041-SP0)
        python: 3.8.5
        datetime: 2022-01-13 11:47:05-0600
        cs_tools: 1.2.1
```

### Example Command

`source unix_install.sh`

### Description

Trying to install on my local machine and finding that `cffi==1.14.5` is failing

**Comment by boonhapus** [comment]
This is a `pip` issue with local installs on Mac. We need to depend on `pip>=20.3`.

> **[Features](https://pip.pypa.io/en/stable/news/#v20-3)**
> - Add support for PEP 600: Future ‘manylinux’ Platform Tags for Portable Linux Built Distributions. (#9077)
> - The new resolver now resolves packages in a deterministic order. (#9100)
> - Add support for MacOS Big Sur compatibility tags. (#9138)

**Comment by boonhapus** [comment]
🐛 Fixed in [v1.3.0](https://github.com/thoughtspot/cs_tools/releases/tag/v1.3.0).
